cluster-diagnosis/__main__.py

```diff
@@ -28,6 +28,15 @@
 exit_code = 0
 
 
+def getBool(v):
```

Comment:
you might want to use distutils instead of this.
https://docs.python.org/2/distutils/apiref.html?highlight=distutils.util#distutils.util.strtobool

Reply:
good point. done.